src/components/AddUser.tsx

```diff
@@ -0,0 +1,280 @@
+    }
+    render() {
+        const {
+            email,
+            name,
+            password,
+            role,
+            cpf,
+            birthDate
+        } = this.state;
+        return (
+            <Mutation
+                mutation={CREATE_OPERATION}
+                variables={{
+                    email,
+                    name,
+                    password,
+                    role,
+                    cpf,
+                    birthDate
+                }}
+                onCompleted={this.handleCreateSuccess}
+            >
+                {(mutation: MutationFn<any>, result: MutationResult) => {
+                    if (result.loading) return (
+                        <div className='sweet-loading' style={{ textAlign: 'center', display: 'block' }}>
+                            <ClipLoader
+                                sizeUnit={"px"}
+                                size={150}
+                                color={'#e6b3ff'}
+                                loading={result.loading}
+                            />
+                        </div>
+                    )
+
+                    return (
+                        <>
+                            <form className="Login" onSubmit={(event) => this.submit(mutation, event)}>
+                                <h1>
+                                    Preencha os dados do novo usuário
+                    </h1>
+                                <div className="panel panel-default">
+                                    <FormErrors formErrors={this.state.formErrors} />
+                                </div>
+                                {result.error && <div style={{ textAlign: 'center', color: 'red' }}>{"Erro!" + result.error.message}</div>}
+
+                                <div className={`form-group ${this.errorClass(this.state.formErrors.email)}`}>
+                                    <label htmlFor="email">E-mail</label>
+                                    <input type="email" required className="form-control" name="email"
+                                        value={this.state.email}
+                                        onChange={this.handleUserEmail} />
+                                </div>
+
+                                <div className={`form-group ${this.errorClass(this.state.formErrors.password)}`}>
+                                    <label htmlFor="password">Senha</label>
+                                    <input type="password" className="form-control" name="password"
+                                        value={this.state.password}
+                                        onChange={this.handleUserPassword} />
+                                </div>
+
+                                <div className={`form-group ${this.errorClass(this.state.formErrors.password)}`}>
+                                    <label htmlFor="password">Nome</label>
+                                    <input  className="form-control" name="name"
```

Comment:
Isso tá errado né? O form é de nome mas o `htmlFor` está password. Assim como o `${this.errorClass(this.state.formErrors.password)`

Reply:
Isso acontece com os forms abaixo desse também.

Reply:
Tá errado mesmo, vou mudar.